pootle/apps/pootle_project/views.py

```diff
@@ -70,12 +70,18 @@ def get_object(self):
                self.kwargs['dir_path']))
 
         if not self.kwargs["filename"]:
-            dirs = Directory.objects.live().select_related("parent__pootle_path")
+            dirs = Directory.objects.live()
+            if self.kwargs['dir_path'].count("/"):
```

Comment:
Perhaps I am wrong, but no class inheriting from `ProjectMixin` actually accepts any call that provides a filename neither a `dir_path`, so is this code actually being run? Is it necessary the `if not self.kwargs["filename"]:` above?